composer.json

```diff
@@ -25,7 +25,7 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
```

Comment:
why "~8.0.0" not "^8.0.0" or just "^8.0"?

Reply:
version constraints defined in https://github.com/laminas/laminas-crypt/issues/10

Reply:
I found also this answer:
- https://github.com/laminas/laminas-mail/issues/114#issuecomment-703315420